codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -260,6 +272,22 @@ public final DefaultStream parent() {
         }
 
         @Override
+        public final boolean isViable() {
```

Comment:
Hah..."writable" in the flow controller context....maybe we can add a chapter in @normanmaurer's book explaining what we mean :)